apps/parsley/src/test_utils/index.tsx

```diff
@@ -95,6 +95,41 @@ const renderWithRouterMatch = (
   };
 };
 
+/**
+ * `overwriteFakeTimers` is a workaround for a bug in @testing-libray/react.
+ * It prevents Vitest's fake timers from functioning with user-event.
+ * https://github.com/testing-library/react-testing-library/issues/1197
+ * @returns an anonymous function overwriting jsdon's Jest field
```

Comment:
nit. (I think these were typos)
```suggestion
 * `overwriteFakeTimers` is a workaround for a bug in @testing-library/react.
 * It prevents Vitest's fake timers from functioning with user-event.
 * https://github.com/testing-library/react-testing-library/issues/1197
 * @returns an anonymous function overwriting jsdom's Jest field
```